### PR TITLE
Mexican restaurant page

### PR DESCRIPTION
**Pull request by blam-codes**

### CODE REVIEW

File: mexican-restaurant-page.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                    <li><a href="#Burrito">Burrito, Bowels & Quesadillas</a></li>
+                    <li><a href="#Tacos">Tacos</a></li>
+                    <!-- salate not sure if its a typo but that is what is on site -->
+                    <li><a href="#Salate">Salate</a><li>
+                    <li><a href="#Dessert">Dessert</a></li>
+                    <li><img src="#" alt="logo"></li>
+                    <!-- styling that when click on phone number gives you pop up option to call -->
+                    <li><a href="url">089 27373381</p><li>
+                    <!-- before e is a special character -->
+                    <li><a href="url google map">ElisabethstraBe 12, 80796</a></li>
```
Comment (dlamdevelops) [inline mexican-restaurant-page.html:57]: This isn't a valid href value.  Most of your href values seem to be placeholder anyways, but this one especially is not correct cause it has spaces.  Can you just add a value of `url` here instead, like line 55?

File: mexican-restaurant-page.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="column">
+                <div class="social">
+                        <h4>News</h4>
+                <!-- CSS to make social links line up with News -->
+                        <a href="url">Facebook</a>
+                        <a href="url">Instagram</a>                       
+                </div>
```
Comment (dlamdevelops) [inline mexican-restaurant-page.html:73]: Indentation issue

File: mexican-restaurant-page.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            <!-- between Happy and Day is an png of burrito -->
+            <p>Happy day</p>
+            <img src="#" alt="picture of three Tacos">
+            <p>Taco Mania Mix & Match @iss.es#newonthemenu #MixIT #mixyourownchoice</p>
+            </div>
+        </div>
+    </div>
+    </div>
+    </div>
+    </div>
```
Comment (dlamdevelops) [inline mexican-restaurant-page.html:88]: Your ending and closing divs are off entirely for these lines.

**Review by dlamdevelops — CHANGES_REQUESTED** [review]
Please see comments.
For any new type of error/issue, please add to your list so that OpenText (your next PR) does not have them.  There are still minor issues with indentation and major issue with opening/closing divs near the end of the document.

File: mexican-restaurant-page.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+                    <li><a href="url">ElisabethstraBe 12, 80796</a></li>
+                    <li>Munchen</li>
+                    <li>Di-Do 18:00-00:00</li>
+                    <li>Fr-Sa: 18:00-02:00</li>
+                </ul>
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row">
```
Comment (dlamdevelops) [inline mexican-restaurant-page.html:66]: Missing closing tag

File: mexican-restaurant-page.html [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+            <div class="column">
```
Comment (dlamdevelops) [inline mexican-restaurant-page.html:67]: Missing closing tag

File: mexican-restaurant-page.html [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+                <div class="social">
+                    <h4>News</h4>
+                <!-- CSS to make social links line up with News -->
+                    <a href="url">Facebook</a>
+                    <a href="url">Instagram</a>                       
+                </div>
+            <br>
```
Comment (dlamdevelops) [inline mexican-restaurant-page.html:74]: Can you explain why `<br>` is here?

File: mexican-restaurant-page.html [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+                <img src="#" alt="picture of burrito on a plate">
+                <!-- between Happy and Day is an png of burrito -->
+                <p>Happy day</p>
+                <img src="#" alt="picture of three Tacos">
+                <p>Taco Mania Mix & Match @iss.es#newonthemenu #MixIT #mixyourownchoice</p>
+            </div>
+        </div>
+    </div>            
+</body>
+
```
Comment (dlamdevelops) [inline mexican-restaurant-page.html:87]: Missing closing `<html>` tag

**Review by dlamdevelops — CHANGES_REQUESTED** [review]
Please see comments and add anything new issue/error to your list.

File: mexican-restaurant-page.html (hunk 6 above)
Comment (blam-codes) [reply mexican-restaurant-page.html:74]: There is a space between social and images below. Looking into br I see this is not the place to use it. Padding would be better. I will incorporate padding when we go into CSS sprint.

File: mexican-restaurant-page.html (hunk 7 above)
Comment (blam-codes) [reply mexican-restaurant-page.html:87]: I don’t have an opening HTML tag. My understanding was if you use <!DOCTYPE html>, you do not have to have <HTML>. As <!DOCTYPE html> is a declaration tag it does not have to be closed.

Comment (dlamdevelops) [reply mexican-restaurant-page.html:87]: Line 1 is: `<!DOCTYPE html>`, which needs to be closed with a closing tag.  
Please see below:
https://www.w3schools.com/tags/tag_doctype.asp

Normally self-closing tags like `<p/>` don't require a closing tag.
However, in this case, `<!DOCTYPE html>` is not self-closing.

**Review by dlamdevelops — CHANGES_REQUESTED** [review]

**Review by dlamdevelops — APPROVED** [review]

**Pull request merged**